app/controllers/users_controller.rb

```diff
@@ -30,8 +42,6 @@ def destroy
     redirect_to :back
   end
 
-  private
-
   def user_params
```

Comment:
- [**Suggestion**] In your user parameter, you will need to add all these below, or you might get some errors: :point_down: 

```
def user_params
  params.require(:user).permit(:name, :email, :password, :password_confirmation)
end
```